pkg/libvirttools/virtualization.go

```diff
@@ -277,6 +280,58 @@ func (v *VirtualizationTool) addVolumesToDomain(podID, podName string, domain *l
 	return nil
 }
 
+func vmLogToFile() bool {
+	if os.Getenv("VIRTLET_VM_LOG_TO_FILE") != "" {
+		return true
+	}
+	return false
+}
+
+func (v *VirtualizationTool) RemoveLibvirtSandboxLog(sandboxId string) error {
+	return os.RemoveAll(filepath.Join(vmLogBaseDir, sandboxId))
+}
+
+func (v *VirtualizationTool) addSerialDevicesToDomain(sandboxId string, domain *libvirtxml.Domain, settings VirtletDomainSettings) error {
+	if settings.vmLogToFile {
+		logDir := filepath.Join(vmLogBaseDir, sandboxId)
+		logPath := filepath.Join(logDir, vmLogFileName)
+
+		// Prepare directory where libvirt will store log file to.
+		if err := os.Mkdir(logDir, 0777); err != nil {
```

Comment:
Is it safe to work with `777`? If it has to be, please use `os.ModePerm`: https://golang.org/pkg/os/#FileMode